Extensions/3D/CustomRuntimeObject3DRenderer.ts

```diff
@@ -131,5 +132,46 @@ namespace gdjs {
     setLayerIndex(layer: gdjs.RuntimeLayer, index: float): void {
       // Layers are not handled for 3D custom objects.
     }
+
+    static getAnimationFrameTextureManager(
+      imageManager: gdjs.PixiImageManager
+    ): ThreeAnimationFrameTextureManager {
+      if (!gdjs.CustomRuntimeObject3DRenderer._animationFrameTextureManager) {
+        gdjs.CustomRuntimeObject3DRenderer._animationFrameTextureManager = new ThreeAnimationFrameTextureManager(
+          imageManager
+        );
+      }
+      return gdjs.CustomRuntimeObject3DRenderer._animationFrameTextureManager;
+    }
+  }
+
+  class ThreeAnimationFrameTextureManager
+    implements gdjs.AnimationFrameTextureManager<THREE.Material> {
+    private _imageManager: gdjs.PixiImageManager;
+
+    constructor(imageManager: gdjs.PixiImageManager) {
+      this._imageManager = imageManager;
+    }
+
+    getAnimationFrame(imageName: string) {
+      return this._imageManager.getThreeMaterial(imageName, {
+        useTransparentTexture: true,
+        forceBasicMaterial: true,
```

Comment:
Yes, it could be interesting to have light and shadow on 3D sprites, but it won't look great without normal map so I guess it's an issue to deal later.